protocol/avdl/keybase1/avatars.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("keybase.1")
+protocol avatars {
+    import idl "common.avdl";
+
+    @typedef("string") record AvatarUrl {}
+    @typedef("string") record AvatarFormat {}
+
+    record LoadAvatarsRes {
+        map<string, map<AvatarFormat, AvatarUrl>> picmap;
```

Comment:
no guarantees if deep-copying this thing actually works.

Reply:
Hah! Never called, so we should be safe.